toobusy.js

```diff
@@ -132,6 +140,11 @@ function start() {
     // Dampen lag. See SMOOTHING_FACTOR initialization at the top of this file.
     currentLag = smoothingFactor * lag + (1 - smoothingFactor) * currentLag;
     lastTime = now;
+
+    if (currentLag > 0) {
```

Comment:
Could you have this set a flag when the first listener is added (and skip the emit if not) so this doesn't fire every single tick? `currentLag` will probably almost always be above 0.